ports/geckolib/glue.rs

```diff
@@ -1530,6 +1530,7 @@ pub unsafe extern "C" fn Servo_SelectorList_Matches(
         None,
         element.owner_document_quirks_mode(),
     );
+    context.scope_element = Some(element.opaque());
```

Comment:
Please fix this too in `components/script/dom/element.rs`?

Reply:
I... just tried... but I don't know how to fix it there... `dom::script::Element` doesn't implement `selectors::Element` so I cannot just call `self.opaque()` there, and it's not clear to me how can I get `DomRoot<Element>` from `self` there.

Reply:
Seems that `DomRoot::from_ref(self)` is what I need here... but you definitely should recheck that I'm doing it the right way.